### PR TITLE
Include the Bullet Chart as a new chart on Britecharts React

### PR DESCRIPTION
**Pull request by Golodhros**
## Description

Adds the Bullet chart from Britecharts

## Motivation and Context

Extending the library

## How Has This Been Tested?

Checked in demos

## Screenshots (if appropriate):
Bullet_—_Britecharts_React_-_React_charting_library_based_in_Britecharts

## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] Latest master code has been merged into this branch
- [ ] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [ ] Build is successful
- [ ] Updated the documentation
- [ ] Added tests to cover changes
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by Golodhros** [comment]
@all-contributors please add @davegomez for code, ideas, maintenance, review

**Comment by Golodhros** [comment]
@all-contributors please add @jchen-eb for code, ideas, maintenance, review

File: src/charts/bullet/Bullet.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import bullet from './bulletChart';
+import {loadingContainerWrapper} from '../loading/LoadingContainer';
+
+class Bullet extends React.Component {
+
+    static propTypes = {
+        /**
+         * Internally used, do not overwrite.
+         */
+        data: PropTypes.array,
```
Comment (jchen-eb) [inline src/charts/bullet/Bullet.js]: Should this one be `isRequired`? Should some of the other props be `isRequired`?

File: src/charts/template/Component.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ class {{pascalCase componentName}} extends React.Component {
         /**
          * TODO: Add your propTypes here!
          */
+        /**
+         *
+         */
+        : PropTypes.,
```
Comment (jchen-eb) [inline src/charts/template/Component.js]: ?

Comment (Golodhros) [reply src/charts/template/Component.js]: This is to make it easy to add the proptypes and comments (note this is the template for the charts)

File: src/charts/bullet/Bullet.js (hunk 1 above)
Comment (Golodhros) [reply src/charts/bullet/Bullet.js]: Good call, I am going to review them all at once.

**Pull request merged**